test/e2e/tests/custom-token-send-transfer.spec.js

```diff
@@ -0,0 +1,414 @@
+        await driver.waitForSelector({ text: 'Create Token', tag: 'button' });
+        await driver.clickElement({ text: 'Create Token', tag: 'button' });
+
+        let windowHandles = await getWindowHandles(driver, 3);
+        await driver.switchToWindow(windowHandles.popup);
+
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+        await driver.switchToWindow(windowHandles.dapp);
+
+        await driver.switchToWindow(windowHandles.dapp);
+
+        await driver.waitForSelector({
+          css: '#tokenAddress',
+          text: '0x',
+        });
+
+        await driver.clickElement({
+          text: 'Add Token to Wallet',
+          tag: 'button',
+        });
+
+        windowHandles = await getWindowHandles(driver, 3);
+        await driver.switchToWindow(windowHandles.popup);
+        await driver.clickElement({ text: 'Add Token', tag: 'button' });
+
+        await driver.switchToWindow(windowHandles.dapp);
+
+        await driver.clickElement({ text: 'Transfer Tokens', tag: 'button' });
+        windowHandles = await getWindowHandles(driver, 3);
+        await driver.switchToWindow(windowHandles.popup);
+
+        await driver.delay(4000);
+        await driver.waitForSelector({ text: 'Edit', tag: 'button' });
+        await driver.clickElement({ text: 'Edit', tag: 'button' });
+
+        await driver.clickElement(
+          { text: 'Edit suggested gas fee', tag: 'button' },
+          10000,
+        );
+
+        const inputs = await driver.findElements('input[type="number"]');
+        const gasLimitInput = inputs[0];
+        const gasPriceInput = inputs[1];
+        await gasLimitInput.fill('60000');
+        await gasPriceInput.fill('10');
+
+        await driver.clickElement({ text: 'Save', tag: 'button' });
+
+        await driver.findElement({ tag: 'span', text: '0.0006' });
+
+        const tokenAmount = await driver.findElement(
+          '.confirm-page-container-summary__title-text',
+        );
+        const tokenAmountText = await tokenAmount.getText();
+        assert.equal(tokenAmountText, '1.5 TST');
+
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+
+        await driver.switchToWindow(windowHandles.extension);
+
+        await driver.clickElement({ tag: 'button', text: 'Activity' });
+        await driver.waitForSelector({
+          css:
+            '.transaction-list__completed-transactions .transaction-list-item__primary-currency',
+          text: '-1.5 TST',
+        });
+
+        await driver.waitForSelector({
+          css: '.list-item__heading',
+          text: 'Send TST',
+        });
+
+        await driver.clickElement({
+          text: 'Assets',
+          tag: 'button',
+        });
+
+        await driver.waitForSelector({
+          css: '.asset-list-item__token-button',
+          text: '8.5 TST',
+        });
+
+        await driver.clickElement({
+          text: 'Activity',
+          tag: 'button',
+        });
```

Comment:
Any reason why we are ending the test with an action instead of an assertion? Why do we need to click on this element?